cmd/ui/src/views/Explore/GraphView.tsx

```diff
@@ -135,12 +135,20 @@ const GraphView: FC = () => {
         </Link>
     );
 
+    const sampleDataLink = (
+        <Link
+            target='_blank'
```

Comment:
Should we add the little "open in new tab" icon to this link?

Reply:
If we do, the "Data Collection" link should get the same treatment.

Reply:
Makes sense to me, should we consult UI / UX?

Reply:
We aren't sure if we have an icon for this in the project yet. Punting for now, but we could always add a ticket later to look at adding an icon to all of our `target='_blank'`